local-cluster/src/local_cluster.rs

```diff
@@ -63,6 +69,52 @@ use {
     },
 };
 
+pub fn build_tpu_quic_client(
+    cluster: &LocalCluster,
+) -> Result<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>> {
+    build_tpu_client(cluster, |rpc_url| Arc::new(RpcClient::new(rpc_url)))
+}
+
+pub fn build_tpu_quic_client_with_commitment(
+    cluster: &LocalCluster,
+    commitment_config: CommitmentConfig,
+) -> Result<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>> {
+    build_tpu_client(cluster, |rpc_url| {
+        Arc::new(RpcClient::new_with_commitment(rpc_url, commitment_config))
+    })
+}
+
+fn build_tpu_client<F>(
+    cluster: &LocalCluster,
```

Comment:
This seems like it should be a method on LocalCluster.

Reply:
ya that's a good point. will update